tools/scrumcheck/ui/assets/app.js

```
@@ -0,0 +1,1066 @@
+                '<div><strong>#' + escHTML(it.Number) + ' - ' + escHTML(it.Title) + '</strong></div>' +
+                '<div><a href="' + escHTML(it.URL) + '" target="_blank" rel="noopener noreferrer">' + escHTML(it.URL) + '</a></div>' +
+                '<ul><li>Status: ' + escHTML(it.Status || '(unset)') + '</li><li>Repository: ' + escHTML(it.Repo || '(unknown)') + '</li><li>Assignees: ' + escHTML(listOrEmpty(it.Assignees, '(none)')) + '</li><li>Labels: ' + escHTML(listOrEmpty(it.Labels, '(none)')) + '</li></ul>' +
+              '</article>'
+            )).join('');
+          return '<div class="project"><h3>' + escHTML(query.Title || '(untitled query)') + '</h3><p class="subtle"><strong>Query:</strong> <code>' + escHTML(query.Query || '') + '</code></p>' + itemsHTML + '</div>';
+        }).join('');
+        setTabClean('generic-queries', total === 0);
+      }
+
+      // Fetch bridge-backed state and re-render all dynamic panels in one pass.
+      async function fetchStateAndRender(forceRefresh) {
```

ESLint

[error] Delete `····`

(prettier/prettier)